fronts-client/src/components/App.tsx

```diff
@@ -139,28 +141,44 @@ const BackgroundHeader = styled.div`
 	width: 100%;
 `;
 
-const App = () => (
-	<ThemeProvider theme={styleTheme}>
-		<DropDisabler>
-			<BannerNotification />
-			<AppContainer>
-				<BackgroundHeader>
-					<SectionHeaderWithLogo greyHeader={true} />
-				</BackgroundHeader>
-				<Switch>
-					<Route {...frontsEditPathProps} component={FrontsEdit} />
-					<Route {...issuePathProps} component={FrontsEdit} />
-					<Route {...frontsFeatureProps} component={FeaturesView} />
-					<Route exact path="/" component={Home} />
-					<Route exact path={manageEditions} component={ManageView} />
-					<Route component={NotFound} />
-				</Switch>
-			</AppContainer>
-			<PlaceholderAnimation />
-			<AppFonts />
-			<OptionsModal />
-		</DropDisabler>
-	</ThemeProvider>
-);
+function usePixelTracking() {
+	const location = useLocation();
+	React.useEffect(() => {
+		if (pageConfig.telemetryUrl) {
+			void fetch(`${pageConfig.telemetryUrl}&path=${location.pathname}`, {
+				credentials: 'include',
+			}).catch(() => {
+				console.error('Fail to send pixel tracking to telemetry service.');
+			});
```

Comment:
Nothing wrong with making the call in this way, but making a request via an image element may be a simpler way to make a dumb GET call.